lib/commands/option-model.json

```diff
@@ -11,6 +11,12 @@
     "alias": null,
     "stringInput": "NONE"
   },
+  "component-name": {
+    "name": "component-name",
+    "description": "Name of the component to be installed.",
```

Comment:
not sure where we landed on terminology but component here seems very generic.

Reply:
suggest to use whatever word nomenclature used elsewhere like  `Alexa Skill reusable component`...